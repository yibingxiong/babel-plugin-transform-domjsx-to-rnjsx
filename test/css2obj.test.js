let postcss = require('postcss');
const fs = require('fs');
const path = require('path');
const plugin = require('../src/css2obj');
var example = fs.readFileSync(path.join(__dirname, '__fixtures__', 'cssSource.css'), {encoding: 'utf8'});

it('css2objtest', async () => {

  let result = await postcss([plugin({plugin})]).process(example, { from: undefined })
  expect(result.css).toMatchSnapshot();
});