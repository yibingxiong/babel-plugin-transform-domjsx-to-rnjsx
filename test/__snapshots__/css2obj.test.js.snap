// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`css2objtest 1`] = `
".className {
	background-color: red;
}
.className2 {
    margin: 10px 20px;
}
"
`;
